README.rst

```diff
@@ -19,6 +19,8 @@ Hopefully this is enough to get you going::
     -v, --verbose
     -d, --dump
     -u UUID, --uuid=UUID  Camera UUID
+    -i id, --id=id        Camera Connection id
+    -c, --connect_with_id Connect using id instead of UUID (NVR Beta API)
```

Comment:
We should be able to detect the newer version somehow and always use the id if necessary, right? Do you know if the older versions allow use of the _id value in place of the uuid? If so, it might be easier to just do that.